backend/social_connections/models.py

```
@@ -96,6 +94,126 @@ class Meta:
+    Inbound rows (direction='in') are a permanent conversation log. Outbound
+    rows are either interactive replies recorded after the fact, or
+    notification deliveries enqueued as status='pending' and drained by the
+    cron-triggered delivery endpoint.
+    """
+
+    DIRECTION_IN = 'in'
+    DIRECTION_OUT = 'out'
+    DIRECTION_CHOICES = [
+        (DIRECTION_IN, 'Inbound'),
+        (DIRECTION_OUT, 'Outbound'),
+    ]
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)